yarn-project/end-to-end/src/e2e_p2p/slashing.test.ts

```diff
@@ -127,6 +131,14 @@ describe('e2e_p2p_slashing', () => {
     t.ctx.aztecNodeConfig.validatorReexecute = false;
     t.ctx.aztecNodeConfig.minTxsPerBlock = 0;
 
+    // Jump forward to an epoch in the future such that the validator set is not empty
+    const slotsInEpoch = await rollup.getEpochDuration();
+    const epochToJumpInto = 4n;
```

Comment:
Why 4? 👀  Hardcoded magic number makes me scared.

Reply:
2 would make sense, but since the slot is the same length as an epoch here, itll actually end up jumping into the past by the time we get here and fail. 
3 is often enough time, but time scared me, so I've set 4